### PR TITLE
Tweak lesson page

### PR DESCRIPTION
**Pull request by arun1595**
Tasks done:

1. Add tweaks to the lesson page
2. Cleaned up lesson completions stuffs

### CODE REVIEW

File: app/views/lessons/show.html.erb [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
 
-<!-- progress-tracking bar -->
-<%= render 'shared/pt_bar', lesson: @lesson %>
-
-<div class="individual-lesson <%= @lesson.is_project ? 'project-lesson' : ''%>">
+<div class="individual-lesson">
   <div class="container">
+    <div class="row">
+      <div class="col-lg-10 offset-lg-1">
+        <%= render 'courses/course/banner', course: @course, show_completion_percent: true %>
```
Comment (KevinMulhern) [inline app/views/lessons/show.html.erb]: I'm not sure if we will be using the percentage completed component on lesson pages. It isn't in the mock ups at least. But lets leave it to see how it looks and functions.

File: app/controllers/lesson_completions_controller.rb [hunk 2]
```diff
@@ -4,12 +4,11 @@ class LessonCompletionsController < ApplicationController
 
   def create
```
Comment (KevinMulhern) [inline app/controllers/lesson_completions_controller.rb]: Do the specs need updated for the changes made in this file?

File: app/helpers/lessons_helper.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module LessonsHelper
+  def course_url(lesson)
```
Comment (KevinMulhern) [inline app/helpers/lessons_helper.rb]: Do we need this in a helper? it seems to be used only in one place

File: app/helpers/lessons_helper.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    link_to 'View Course',
+      course_path(lesson.course.title_url, ref: 'lnav'),
+      title: "Go back to '#{lesson.course.title}'",
+      class: 'button button--secondary'
+  end
+
+  def next_lesson_url(lesson)
```
Comment (KevinMulhern) [inline app/helpers/lessons_helper.rb]: Same for this one?

**Comment by KevinMulhern** [comment]
Thanks @arun1595, this is looking really good 😃

File: app/controllers/lesson_completions_controller.rb (hunk 2 above)
Comment (arun1595) [inline app/controllers/lesson_completions_controller.rb]: Yeah, thanks for pointing it out.

Let me do it.

File: app/helpers/lessons_helper.rb (hunk 3 above)
Comment (arun1595) [inline app/helpers/lessons_helper.rb]: You're right. The view was a bit cluttered, the only motivation to move this to the helper was to make the view code readable :smile:

File: app/controllers/lesson_completions_controller.rb (hunk 2 above)
Comment (arun1595) [inline app/controllers/lesson_completions_controller.rb]: Done. PTAL

**Comment by KevinMulhern** [comment]
👍 LGTM
Thanks @arun1595 😄

**Pull request merged**